src/imitation/algorithms/adversarial/common.py

```diff
@@ -421,7 +422,7 @@ def train_gen(
     def train(
```

Comment:
One more thing - if you change the arguments, update of [training_adversarial.py](https://github.com/gunnxx/imitation/blob/7b3ed752bae1d48af7c4e5d6be06a3447b39c713/src/imitation/scripts/train_adversarial.py#L154) will also be needed